src/vs/workbench/contrib/welcomeGettingStarted/browser/positronWelcomePageHelp.tsx

```diff
@@ -4,32 +4,47 @@
  *--------------------------------------------------------------------------------------------*/
 
 // CSS.
-import 'vs/css!./media/positronGettingStarted';
```

Comment:
Could you please revert the double quote usage as we use single quotes for ts strings. The double quotes will raise warnings complaining they should only be used on externalized strings.

Reply:
Yep, fixed!